scenariolib/visit.go

```diff
@@ -67,23 +67,19 @@ func NewVisit(_searchtoken string, _uatoken string, _useragent string, language
 
 	v.WaitBetweenActions = !c.DontWaitBetweenVisits
 	v.Anonymous = false
-	if c.AllowAnonymous {
```

Comment:
This change breaks scenarios that only had `AllowAnonymous` and expected the default threshold.

Do you know if some people could have a scenario outside of the repository with this? 

So:

1. If you don't care, ignore (totally valid option in this case, IMO :)).
2. If you want to be compatible, at least detect the property and warn to not use it anymore.